website/views/shared/modals/members.jade

```diff
@@ -46,9 +46,12 @@ script(type='text/ng-template', id='modals/private-message.html')
   .modal-header
     h4=env.t('pmHeading', {name: "{{profile.profile.name}}"})
   .modal-body
-    textarea.form-control(type='text',rows='5',ui-keydown='{"meta-enter":"sendPrivateMessage(profile._id, _message)"}',ng-model='_message')
+    textarea.form-control(type='text',rows='5',ui-keydown='{"meta-enter":"sendPrivateMessage(profile._id, _message)"}',placeholder=env.t('needsTextPlaceholder'),ng-model='_message')
   .modal-footer
-    button.btn.btn-primary(ng-click='sendPrivateMessage(profile._id, _message)')=env.t("send")
+    //- Due to a quirk in Bootstrap UI, we need to wrap this so the tooltip can
+    //- appear when disabled.
+    div(style='display:inline-block',tooltip="{{_message ? '' : env.t('needsText')}}")
```

Comment:
I'm going to trust you on the need to wrap it. Is there a reason it needs to be `display: inline-block` instead of `inline`? If it can just be inline, you could change it to a span and remove the inline style. If it must be `inline-block`, I'd rather you [use the bootstrap class](http://getbootstrap.com/css/#responsive-utilities-classes) for doing inline styles:

```
.visible-xs-inline-block(tooltip="{{_message ? '' : env.t('needsText')}}")
```